scout/server/static/bs_styles.css

```diff
@@ -33,7 +33,15 @@ table.dataTable {margin-bottom: 0}
 }
 
 a:visited {
-	color: var(--bs-purple);
+	color: var(--bs-cyan);
```

Comment:
Actually you could revert to purple now after fixing the button links..

Reply:
Yes, but perhaps cyan is more visible than purple?

Purple:

<img width="209" alt="image" src="https://user-images.githubusercontent.com/28093618/173583538-33be6b4d-ac0d-4259-862a-9cdeb387a02b.png">


Cyan:

<img width="208" alt="image" src="https://user-images.githubusercontent.com/28093618/173583597-69a3dfa6-edf0-42f7-a7c5-295d91a152ab.png">

Maybe not as much on white tho:

<img width="195" alt="image" src="https://user-images.githubusercontent.com/28093618/173583747-9abe9eed-ce42-4e54-9183-b7c797728152.png">